docs/source/tracking.rst

```diff
@@ -934,6 +1039,47 @@ You can then pass authentication headers to MLflow using these :ref:`environment
 Additionally, you should ensure that the ``--backend-store-uri`` (which defaults to the
 ``./mlruns`` directory) points to a persistent (non-ephemeral) disk or database connection.
 
+.. _artifact_only_mode:
+
+Using the Tracking Server exclusively for proxied artifact access
+-----------------------------------------------------------------
+
+To use an instance of the MLflow tracking server *exclusively* for artifact operations ( :ref:`scenario_6` ),
+start a server with the optional parameters ``--serve-artifacts`` to enable proxied artifact access and ``--artifacts-only``
+for disabling all other functionality of the tracking server.
+
+To start the MLflow server in this restricted mode with proxied access to an HDFS location (as an example):
+
+.. code-block:: bash
+
+    mlflow server \
+        --host 0.0.0.0 \
+        --port 8885 \
+        --artifacts-destination hdfs://myhost:8887/mlprojects/models \
+        --serve-artifacts \
+        --artifacts-only
+
+Using a tracking server configured in ``--artifacts-only`` mode for any tasks aside from those concerned with artifact
+handling (i.e., model logging, loading models, logging artifacts, listing artifacts, etc.) will return an HTTPError.
+See the following example of a client REST call in Python attempting to list experiments from a server that is configured in
+``--artifacts-only`` mode:
+
+.. code-block:: python
+
+    import requests
+    response = requests.get("http://0.0.0.0:8885/api/2.0/mlflow/experiments/list")
+
+Output
+
+.. code-block:: text
+
+    >> HTTPError: Endpoint: /api/2.0/mlflow/experiments/list disabled due to the mlflow server running in `--artifacts-only` mode.
+
+Using this mode to control access to artifacts without exposing the server endpoint's ability to create experiments,
+manage runs, or perform any action apart from artifact handling can be useful in some scenarios (continuous deployment
+by an external team, for instance).
```

Comment:
I think the biggest motivator is decoupling the compute / infrastructure used for serving these different types of requests, since each type of request (artifacts vs metadata) has different performance requirements and usage characteristics.